src/test/java/teammates/test/cases/action/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -695,6 +695,95 @@ public void testExecuteAndPostProcess() throws Exception {
                 r.getDestinationWithParams());
         assertNotNull(frDb.getFeedbackResponse(fq.getId(), fr.giver, fr.recipient));
 
+        ______TS("Unsuccessful case: const sum: over maximum points");
+        String maxValueQn1 = "90";
+
+        submissionParams = new String[] {
+                Const.ParamsNames.FEEDBACK_QUESTION_RESPONSETOTAL + "-1", "1",
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID + "-1-0", fr.getId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, fr.feedbackSessionName,
+                Const.ParamsNames.COURSE_ID, fr.courseId,
+                Const.ParamsNames.FEEDBACK_QUESTION_ID + "-1", fr.feedbackQuestionId,
+                Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-1-0", fr.recipient,
+                Const.ParamsNames.FEEDBACK_QUESTION_TYPE + "-1", fr.feedbackQuestionType.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-1-0", "91",
+                Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-1-0", "9",
```

Comment:
Is "9" valid?
We shall only put one invalid test data to test the unsuccessful scenario. It would be better to put `90` in this case (`Boundary Value Analysis`)